libraries/chain/controller.cpp

```diff
@@ -3339,9 +3380,20 @@ struct controller_impl {
       }
 
       guard_pending.cancel();
+
       return onblock_trace;
    } /// start_block
 
+   void run_readonly_transactions(fc::time_point deadline) {
```

Comment:
I think a better name would be something like: `update_peer_keys()`

Reply:
I don't mind changing the name, but the reason I named it like that was because I though we may use other read-only transactions in the future to access chainbase data, so I thought better to use a non-specific name.

Reply:
My vote would be to give it a more descriptive name.

Reply:
https://github.com/AntelopeIO/spring/pull/1316/commits/9ca911a165eb9bafeb2a1309ffc9fabf6ad70e15 hopefully not too much!

Reply:
I don't see https://github.com/AntelopeIO/spring/commit/9ca911a165eb9bafeb2a1309ffc9fabf6ad70e15 being any better than `run_readonly_transactiions`. I think it should describe more what is accomplishing as opposed to how it is accomplishing it.

Reply:
https://github.com/AntelopeIO/spring/pull/1316/commits/59e93360c2677ef1b15bc0949fd14d8191aed2ee